Multy/Cells/AddressCell/WalletAddressTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//Copyright 2017 Idealnaya rabota LLC
+//Licensed under Multy.io license.
+//See LICENSE for details
+
+import UIKit
+
+class WalletAddressTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var addressLbl: UILabel!
+    @IBOutlet weak var sumLbl: UILabel!             // 0.0001 BTC / 100 USD
+    @IBOutlet weak var creationTimeLbl: UILabel!    // 1.11.2017 ∙  23:15
+    
+    var wallet: UserWalletRLM?
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+    
+    func fillInCell(index: Int) {
+        var sumInCrypto = 0.0
+        var sumInFiat = 0.0
+        self.addressLbl.text = self.wallet?.addresses[index].address
+        
+        if self.wallet?.chain == 0 {
```

Comment:
In my opinion, there should be a dedicated class that knows exactly how many satoshis, wei, whatever in corresponding cryptocurrency unit. And can convert that both ways by `chain` id.
Same thing for fiat value, the only difference is that we should not call that a fiat, since some day we'll use other cryptocurrency here.

Reply:
We will do it later